libcxx/docs/ReleaseNotes/20.rst

```diff
@@ -45,6 +45,7 @@ Implemented Papers
 - ``std::jthread`` and ``<stop_token>`` are not guarded behind ``-fexperimental-library`` anymore
 - P2674R1: A trait for implicit lifetime types (`Github <https://github.com/llvm/llvm-project/issues/105259>`__)
 - P0429R9: A Standard ``flat_map`` is partially implemented and ``flat_map`` is provided (`Github <https://github.com/llvm/llvm-project/issues/105190>`__)
+- P3379R1: Constrain ``std::expected`` equality operators
```

Comment:
The paper number is R0, please adjust all references in the PR
```suggestion
- P3379R0: Constrain ``std::expected`` equality operators
```

Reply:
There was a D3379R1 draft but I forgot to publish it, and we voted on R0. The only difference is that R1 said to add `__cpp_lib_constrained_comparisons` in `<expected>`. That addition got done editorially: https://github.com/cplusplus/draft/pull/7449

It looks like this PR neither bumps the macro value nor adds the macro to `<expected>`.

Reply:
I've submitted https://github.com/brevzin/sd6/pull/28 to update SD-6.